### PR TITLE
APPLE-165: Theme editor to reflect fonts uploaded to News Publisher (release/v2.5.0)

### LINKED ISSUE #1067: Add Custom Fonts Uploaded to Channel to Font List in Theme Editor
### Description

I would like to see any custom fonts uploaded to a channel available for selection in the font dropdown in the theme editor. Additionally, it would be nice to have a warning if the selected font is not available on the user's system, which would alert the user that the preview on the right won't be accurate unless they install that custom font locally.

### Use Case

When a user uses the theme editor, they should be able to select from all available fonts, including custom fonts uploaded to their channel.

### Additional Information

The channel API response should include what custom fonts, if any, are available for the channel.

### PR DESCRIPTION
**Pull request by kaitlinbolling**
## Summary

Updates the Theme editor to reflect fonts uploaded to News Publisher.

### Changed

- Theme editor now reflects fonts uploaded to News Publisher.

Fixes #1067

### CODE REVIEW

File: assets/js/theme-edit.js [hunk 1]
```diff
@@ -23,21 +23,41 @@
...
-					.text( appleNewsThemeEdit.fontNotice )
-			)
-		}
+		$( '.select2.standard' ).select2();
+		$( '.select2.font' ).select2({
+			templateResult: appleNewsFontSelectTemplate,
+			templateSelection: appleNewsFontSelectTemplate
+		}).on('select2:select', async function (e) {
+			// Check if font preview is available.
+			var selectedFont = e.params.data.text;
```
Comment (attackant) [inline assets/js/theme-edit.js:32]: Should we use `const` or `let` here? And in the next few lines?

**Review by attackant — APPROVED** [review]
Approved but with a question for your review 🐙

File: assets/js/theme-edit.js (hunk 1 above)
Comment (kaitlinbolling) [reply assets/js/theme-edit.js:32]: I used `var` to remain consistent with the rest of the file, but just noticed I used `let` in some cases. So I will update these to `const`!

Comment (kevinfodness) [reply assets/js/theme-edit.js:32]: I believe this is _not_ transpiled JS, so I'd recommend having them stay as `var` - the `let` was probably an oversight

Comment (attackant) [reply assets/js/theme-edit.js:32]: It was my understanding these were pretty universally supported now, is that not the case?

Comment (kevinfodness) [reply assets/js/theme-edit.js:32]: They are, but for consistency within this repo and because we aren't transpiling, I'd prefer to use `var`.

**Pull request merged**